contracts/LockedTokens.cdc

```diff
@@ -744,11 +744,6 @@ access(all) contract LockedTokens {
         self.LockedAccountCreatorPublicPath = /public/lockedAccountCreator
 
         /// create a single admin collection and store it
-        admin.save(<-create TokenAdminCollection(), to: self.LockedTokenAdminCollectionStoragePath)
-
-        admin.link<&LockedTokens.TokenAdminCollection>(
-            LockedTokens.LockedTokenAdminPrivatePath,
-            target: LockedTokens.LockedTokenAdminCollectionStoragePath
-        ) ?? panic("Could not get a capability to the admin collection")
```

Comment:
Not sure what was the reason for linking to a private path here.
Removed it for the moment.

Reply:
Probably for handing out a private capability to another account? @joshuahannan

Reply:
yeah, it was for giving a private capability for privileged access to another account

Reply:
The equivalent of linking a private capability is issuing a capability controller and not publishing it.

It depends on the use-case what you want to do here. Are you just asking for a literal equivalent of the current code, or also how the code should maybe be changed to be more in line with the use-case.

For example, the literal translation is issuing a controller, and saving the resulting capability for later use.

However, you might also consider:
- Delaying the issuing, and not saving in storage
- Instead of reusing a link/capability/controller for multiple different purposes/users, issue a new capability/controller each time, so they can be individually controlled, tagged, etc.

Happy to hop on a quick sync about it, might be easier to discuss there

Reply:
Also how critical is this for the preview build? Is this something we can update later?

Reply:
Honestly, we can probably just remove this. I don't even know if it is used in the tests at all and it is in the init statement so it doesn't really matter for what is on mainnet right now